content/en/training/_index.html

```diff
@@ -112,7 +112,6 @@ <h2>Kubernetes Training Partners</h2>
     </center>
   </div>
   <div class="main-section landscape-section">
-    <iframe src="https://landscape.cncf.io/category=kubernetes-training-partner&format=logo-mode&grouping=category&embed=yes" frameborder="0" id="landscape" scrolling="no"></iframe>
-    <script src="https://landscape.cncf.io/iframeResizer.js"></script>
+    {{< cncf-landscape helpers=false $training=true >}}
```

Comment:
@Arhell this looks like it is hard-coding a detail of the training page

How about making the call be:
```markdown
     {{< cncf-landscape helpers=false category="kubernetes-training-partner" >}}
```
and then adjusting the shortcode to use that data?

Reply:
done, check please 
review page
https://deploy-preview-24014--kubernetes-io-master-staging.netlify.app/training/